src/cosmos/thorchainTypes.ts

```diff
@@ -48,11 +50,14 @@ export const asMidgardActionsResponse = asObject({
       metadata: asObject(
         asObject({
           memo: asString,
-          networkFees: asArray(
-            asObject({
-              amount: asString,
-              asset: asString // 'THOR.RUNE'
-            })
+          networkFees: asMaybe(
```

Comment:
Wouldn't it be `asOptional`, if it's simply missing? Otherwise we might hide arbitrary corruptions, such as `amount` being a number or something else. In that case, we would silently delete the whole fees array instead of complaining about the specific thing that's wrong inside.

Reply:
yup, good catch